recipes/resistify/meta.yaml

```
@@ -40,5 +48,12 @@ test:
 
 about:
   home: https://github.com/swiftseal/resistify
+  doc_url: "https://github.com/SwiftSeal/resistify/blob/main/README.md"
   license: MIT
+  license_file: LICENSE
   summary: A resistance gene annotation tool
+  description: |
+    Resistify is a nucleotide-binding leucine rich repeat (NLR) annotation
+    program, for the purpose of identifying putative disease resistance
+    genes in plant genomes.
+  
```

yamllint

[error] trailing spaces

(trailing-spaces)